Tangem/Modules/Welcome/WelcomeTokenList/WelcomeTokenListItemView.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  WelcomeTokenListItemView.swift
+//  Tangem
+//
+//  Created by skibinalexander on 18.09.2023.
+//  Copyright © 2023 Tangem AG. All rights reserved.
+//
+
+import SwiftUI
+
+struct WelcomeTokenListItemView: View {
+    @ObservedObject var model: WelcomeTokenListItemViewModel
+
+    let arrowWidth: Double
```

Comment:
```suggestion
private enum Constants { 
    static let arrowWidth: Double = 46
}
```
так лучше будет
И можно вниз перенести